gulp/tasks/assets.js

```diff
@@ -1,43 +1,27 @@
-const merge = require('merge-stream');
+const path = require('path');
 
 module.exports = (gulp, $, config) => {
-  const imagesSrc = config.appFiles.images;
-  const imagesDest = config.paths.images.dest;
-  const logosSrc = config.appFiles.logos;
-  const logosDest = config.paths.logos.dest;
-  const faviconsSrc = config.appFiles.favicons;
-  const faviconsDest = config.paths.favicons.dest;
-  const fontsSrc = config.appFiles.fonts;
-  const fontsDest = config.paths.fonts.dest;
+  const assetsSrc = config.appFiles.assets;
+  const assetsDest = config.paths.assets.dest;
   const manifestFile = config.paths.revManifest.dest;
 
-  const task = () => {
-    const imagesSt = gulp.src(imagesSrc)
-      .pipe($.changed(imagesDest))
-      .pipe($.if(config.isProd, $.rev()))
-      .pipe($.if(config.isProd, gulp.dest(imagesDest)))
-      .pipe($.if(config.isProd, $.rev.manifest(manifestFile, { merge: true, base: imagesDest })))
-      .pipe(gulp.dest(imagesDest))
-      ;
-
-    const logosSt = gulp.src(logosSrc)
-      .pipe($.changed(logosSrc))
-      .pipe($.if(config.isProd, $.rev()))
-      .pipe($.if(config.isProd, gulp.dest(logosDest)))
-      .pipe($.if(config.isProd, $.rev.manifest(manifestFile, { merge: true, base: logosDest })))
-      .pipe(gulp.dest(logosDest));
+  const blacklistedFolders = [
+    path.normalize(path.resolve(__dirname, '../../', config.paths.content.src)), // /content/texts
```

Comment:
I still prefer this to be explicit.
In the config, list all the folders containing assets to copy instead of having to maintain the extentions plus this blacklist.